(node:4059145) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-session-42311',
  containerIp: '10.0.3.15',
  mitmPort: 42311,
  autoAccessFixed: 1
}
2024-11-08 11:47:44.228 - [Info] MITM Version: 2
2024-11-08 11:47:44.230 - [Info] Auto Access Enabled: true
2024-11-08 11:47:44.231 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-08 11:47:44.256 - [Info] SSH man-in-the-middle server for 10.0.3.15 listening on 10.0.3.1:42311
2024-11-08 11:48:07.766 - [Debug] [Connection] Attacker connected: 36.129.11.110 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 11:48:10.307 - [Debug] [Auth] Attacker 36.129.11.110 trying to authenticate with 'password'
2024-11-08 11:48:10.308 - [Debug] [Auto Access] Attacker: 36.129.11.110, Threshold: 1, Attempts: 1
2024-11-08 11:48:10.309 - [Debug] [Auto Access] Compromising the honeypot
2024-11-08 11:48:10.309 - [Debug] [Auto Access] Adding the following credentials: 'cristina:cristina'
2024-11-08 11:48:10.565 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-08 11:48:10.568 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-08 11:48:10.568 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.15
2024-11-08 11:48:10.644 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-08 11:48:11.980 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-08 11:48:13.932 - [Debug] [Connection] Attacker connected: 36.129.11.110 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 11:48:15.829 - [Debug] [Auth] Attacker 36.129.11.110 trying to authenticate with 'password'
2024-11-08 11:48:15.830 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-08 11:48:15.831 - [Debug] [Auth] Attacker: 36.129.11.110 has so far made 1 attempts. Remaining: 5 attempts
2024-11-08 11:48:16.492 - [Debug] [Connection] Attacker closed the connection
2024-11-08 11:48:16.793 - [Debug] [Connection] Attacker connected: 36.129.11.110 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-08 11:48:17.194 - [Debug] [Connection] Attacker connected: 47.236.21.181 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
